### ISSUE TITLE
afterColumnMove/beforeColumnMove doesn't tell you where the columns move to

### ISSUE THREAD

**Issue by aaronbeall** [handsontable/handsontable#6005]
### Description

Both `afterColumnMove` and `beforeColumnMove` have the signature:

Param | Type | Description
-- | -- | --
`columns` | `Array.<Number>` | Array of visual column indexes to be [or that were] moved.
`target` | `Number` | Visual column index being a target for moved columns.

I found it surprisingly hard to figure out what the actual destination indexes will be, especially when multiple columns are being moved. I came up with:
```
  afterColumnMove(columns: number[], target: number) {

      // https://github.com/handsontable/handsontable/issues/5958
      if (!columns.length) return;

      const movedColumns = columns.map((column, index) =>{
        const destinationIndex = target + index - (target > column ? columns.length : 0);
        const prop = instance.colToProp(destinationIndex);
        return { prop, destinationIndex }
      });

      // Make an API call with { prop, destinationIndex } data
  };
```

I think that's correct?

### Request

It'd be great to have a 3rd argument for both hooks `destinationColumns: number[]` that tells us exactly what indexes the columns are moved to. 

```
afterColumnMove(columns: number[], target: number, destinationColumns: number[]) 
```

### Demo

https://jsfiddle.net/handsoncode/8ffpsqt6/

### Your environment
* Handsontable version: 7.0.2
* Browser Name and version:
* Operating System:

**Comment by AMBudnik** [comment]
I'm glad that you've mentioned this hook. It got a bit messy.

Since forever (`0.28.4`) we're rolling with a [bug with indexes](https://github.com/handsontable/handsontable/issues/4470) and [related](https://github.com/handsontable/handsontable/issues/5173), but this one should be fixed by [the new structure](https://github.com/handsontable/handsontable/pull/5945). But.. that's a huge change...
![image](https://user-images.githubusercontent.com/10757813/57620655-b2aa8600-7589-11e9-92ac-aec0c118fbf1.png)
.. so it will take us some time to test and retouch.

Then the `updateSettings bugs`
https://github.com/handsontable/handsontable/issues/4480
https://github.com/handsontable/handsontable/issues/6004

and.. the topic that you've mentioned about firing the hook event if we aborted the change in the ~before hook.

The hook itself has two parameters where the first one tells us what are the visual indexes of moves columns and how many of those we have (array). While the second parameter tells us where to leave those columns (visual index). And yup... I agree that we would need to perform some additional transformations to get the destination array.
If we're planning to allow moving of nonadjacent columns that would be a must.

**Comment by aninde** [comment]
@aaronbeall we are currently testing a version `8.0.0rc-1` with many breaking changes.

In `8.0.0-rc1` v. @wszymanski  has been rewriting `manualColumnMove`.

Answering your wish, we introduced
https://github.com/handsontable/handsontable/blob/f45d9af027fc92213664d471002cd1ec5870787f/src/plugins/manualRowMove/manualRowMove.js#L160

I recommend observing news from issue #5751. More about changes in the PR #5945.
Changes will be released soon.

**Comment by AMBudnik** [comment]
Hey @aaronbeall 

We've just released the beta for 8.0.0. where we fixed this issue. We want to keep all related issues open and close them after releasing the stable version. 
Any feedback is highly appreciated.

**Comment by AMBudnik** [comment]
I guess that Aaron's being notified by my updates for the nth time now ;D but I guess that knowing that every beep means that another issue is gone is quite pleasant.

![Zrzut ekranu 2020-08-6 o 15 20 39](https://user-images.githubusercontent.com/10757813/89536574-684d9e80-d7f8-11ea-8425-89dfbe4f0b91.png)

![Zrzut ekranu 2020-08-6 o 15 21 37](https://user-images.githubusercontent.com/10757813/89536656-89ae8a80-d7f8-11ea-8c2e-c6b0bf1c3adc.png)
